File: javaapp/src/main/java/com/example/javaapp/MainActivity.java
package com.example.javaapp;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

import com.example.mybutterknife.BindView;
import com.example.mybutterknife.Binding;

public class MainActivity extends AppCompatActivity {

    @BindView(R.id.mTvFirst)
    TextView mTvTextView;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Binding.INSTANCE.bind(this);
        mTvTextView.setText("改变了");
    }
}

File: app/src/main/java/com/example/kaixue/MainActivity.kt
package com.example.kaixue

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView
import com.example.mybutterknife.BindView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    override fun setSupportProgress(progress: Int) {
    }
}